src/coffee/directives/api/models/child/marker-child-model.coffee

```diff
@@ -23,7 +23,7 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
         @trackModel = true, @needRedraw = false) ->
         #where @model is a reference to model in the controller scope
         #clonedModel is a copy for comparison
-        @clonedModel = _.extend({},@model)
+        @clonedModel = _.clone(@model,true)
```

Comment:
I was doing `_.extend {},` prior as it was to be a new total fresh object , thus clone deep. But if this fixes issues then I guess it is good. It is intended that the clone is a complete break from the original reference.